tardis/plasma/properties/level_population.py

```diff
@@ -8,25 +8,23 @@
 
 __all__ = ['LevelNumberDensity', 'LevelNumberDensityHeNLTE']
 
+
 class LevelNumberDensity(ProcessingPlasmaProperty):
     """
     Attributes:
     level_number_density : Pandas DataFrame, dtype float
-                           Index atom number, ion number, level number. Columns are zones.
+                           Index atom number, ion number, level number.
+                           Columns are zones.
     """
     outputs = ('level_number_density',)
     latex_name = ('N_{i,j,k}',)
     latex_formula = ('N_{i,j}\\dfrac{bf_{i,j,k}}{Z_{i,j}}',)
 
-    def calculate(self):
-        pass
-
     def __init__(self, plasma_parent):
         """
         Calculates the level populations with the Boltzmann equation in LTE.
```

Comment:
I know that this is not particularly relevant for this PR but this class has nothing to do with either the Boltzmann equation or LTE.  It is used for all excitation treatments including dilute-lte and nlte.